src/main/java/scrolls/elder/logic/commands/LogAddCommand.java

```diff
@@ -0,0 +1,115 @@
+package scrolls.elder.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static scrolls.elder.logic.parser.CliSyntax.PREFIX_DURATION;
+import static scrolls.elder.logic.parser.CliSyntax.PREFIX_REMARKS;
+import static scrolls.elder.logic.parser.CliSyntax.PREFIX_START;
+
+import java.util.Date;
+import java.util.List;
+
+import scrolls.elder.commons.core.index.Index;
+import scrolls.elder.commons.util.ToStringBuilder;
+import scrolls.elder.logic.Messages;
+import scrolls.elder.logic.commands.exceptions.CommandException;
+import scrolls.elder.model.LogStore;
+import scrolls.elder.model.Model;
+import scrolls.elder.model.PersonStore;
+import scrolls.elder.model.log.Log;
+import scrolls.elder.model.person.Person;
+
+/**
+ * Adds a person to the address book.
+ */
+public class LogAddCommand extends Command {
+
+    public static final String COMMAND_WORD = "logadd";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a log to the address book. "
+        + "Parameters: INDEX1 INDEX2 "
+        + PREFIX_START + "START_DATE (yyyy-MM-dd) "
+        + PREFIX_DURATION + "DURATION (in hours) "
+        + PREFIX_REMARKS + "REMARKS "
+        + "Example: " + COMMAND_WORD + " 1 2 "
+        + PREFIX_START + "2021-03-01 "
+        + PREFIX_DURATION + "2 "
+        + PREFIX_REMARKS + "was a good session";
+
+    public static final String MESSAGE_SUCCESS = "New log added!";
+
+    /**
+     * Contains data for the log to be added.
+     * Does not contain the final log ID.
+     */
+    private final Index volunteerIndex;
+    private final Index befriendeeIndex;
+    private final int duration;
+    private final Date startDate;
+    private final String remarks;
+
+    /**
+     * Creates an LogAddCommand to add the specified {@code Log}
+     */
+    public LogAddCommand(Index volunteerIndex, Index befriendeeIndex, int duration, Date startDate, String remarks) {
+        this.volunteerIndex = volunteerIndex;
+        this.befriendeeIndex = befriendeeIndex;
+        this.duration = duration;
+        this.startDate = startDate;
+        this.remarks = remarks;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+
```

Comment:
Well-written execute method for LogAddCommand with no deep nesting and a highly logical code structure that maintains readability!